### PR TITLE
Setup for the No Filter Results hookup

### PR DESCRIPTION
**Pull request by svanherk**
- Add the svg and rename the other image component

### CODE REVIEW

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
 import {Rels, Classes} from 'd2l-hypermedia-constants';
 import '../d2l-activity-name/d2l-activity-name.js';
 import '../d2l-activity-evaluation-icon/d2l-activity-evaluation-icon-base.js';
-import './d2l-no-submissions-image.js';
+import './d2l-evaluation-hub-no-submissions-image.js';
+import './d2l-evaluation-hub-no-filter-results-image.js';
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: This sin't used in here yet, but adding the import and some styles for when it is

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js [hunk 2]
```diff
@@ -16,7 +16,6 @@ class D2LNoSubmissionImage extends PolymerElement {
 			    <defs>
 			        <style>.cls-1{isolation:isolate;}.cls-2{fill:#f9fafb;}.cls-3{fill:#e6eaf0;}.cls-4{fill:#9ea6b2;}.cls-5{fill:#7c8695;}.cls-6{fill:#b9c2d0;}.cls-7{opacity:0.1;mix-blend-mode:multiply;}.cls-10,.cls-12,.cls-13,.cls-16,.cls-8{fill:none;}.cls-8{stroke:#565a5c;}.cls-10,.cls-12,.cls-13,.cls-8{stroke-miterlimit:10;}.cls-10,.cls-8{stroke-width:0.852px;}.cls-9{fill:#565a5c;}.cls-10{stroke:#72777a;}.cls-11{fill:#fff;}.cls-12{stroke:#9ea6b2;stroke-width:1.122px;}.cls-12,.cls-13,.cls-16{stroke-linecap:round;}.cls-13{stroke:#46a661;stroke-width:2.525px;fill-rule:evenodd;}.cls-14{fill:#d3d9e3;}.cls-15{fill:#a9b1be;}.cls-16{stroke:#7c8695;stroke-linejoin:round;stroke-width:0.838px;}</style>
 			    </defs>
-			    <title>Asset 16</title>
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: This makes a tooltip with "Asset 16" appear in it when we hover over it

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js [hunk 3]
```diff
@@ -78,8 +77,8 @@ class D2LNoSubmissionImage extends PolymerElement {
 		`;
 	}
 
-	static get is() { return 'd2l-no-submissions-image'; }
+	static get is() { return 'd2l-evaluation-hub-no-submissions-image'; }
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: Renaming this to make sure we have super unique component names, in response to the BSI problem today

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
  * @customElement
  * @polymer
  */
-class D2LNoSubmissionImage extends PolymerElement {
+class D2LEvaluationHubNoSubmissionImage extends PolymerElement {
```
Comment (AlexBedley) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: QuickEval? I realize this puts us in a half state but it's less to do once we do the full rename. Or are you thinking we'll never do the rename?

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js (hunk 2 above)
Comment (AlexBedley) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: Why was this here in the first place?

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js (hunk 1 above)
Comment (AlexBedley) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Spoke offline about this, we'll call out in the next PR that we've added some css already

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js (hunk 2 above)
Comment (svanherk) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: It was in what design sent us, I just noticed this today

File: components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js (hunk 4 above)
Comment (svanherk) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-no-submissions-image.js]: Talked offline, I don't know what will happen.  There's a lot to fight for/prioritize in the meeting on Friday.  We're going to just leave it like this for now

**Pull request merged**